### DISCUSSION TITLE
Organize a single folder

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [facebook/docusaurus#8805] [Q&A] [answered]
Hey there 👋,

Is there any way to organize in the sidebar only the files from a certain folder and all other/other folders remain autogenerated?

Example:

```txt
doc/
   intro.md
   other.md
   etc...
   myOrganizedFolder/
     fileOne.md
     secondFile.md
     etc...
```

I know I can put `sidebar_position` at the beginning of each file, but this is too manual and confusing. I would like a more practical way of organizing, without having to change the whole `sidebar.js`.

Sorry if I was too confused 😅

**Comment by slorber** [comment, marked as answer]
I guess what you want could be something like this

`{type: 'autogenerated', exclude: ["./myOrganizedFolder"]}`

You can present your use-case in more details here if it's not already listed: https://github.com/facebook/docusaurus/issues/5689

**Reply by 201flaviosilva** [reply]
Hmm, so does that mean it's not possible, yet? 😅

**Reply by slorber** [reply]
It is currently not possible to autogenerate from the root and exclude one specific folder, unless you customize the sidebar items generator yourself to achieve your need. 

https://docusaurus.io/docs/sidebar/autogenerated#customize-the-sidebar-items-generator

It's not impossible because the generation logic is flexible, but it not convenient and is more work for you to find a solution.

**Reply by 201flaviosilva** [reply]
Yeh, for now I will just change the `sidebar.js`: https://github.com/201flaviosilva-labs/Documentum/blob/main/sidebars.js

Thx for the help 😁
